src/main/java/com/adobe/commerce/cif/model/shoppinglist/ShoppingList.java

```diff
@@ -33,9 +33,6 @@
     @ApiModelProperty(value = "The entries of the shopping list.", required = true)
     protected List<ShoppingListEntry> entries;
 
-    @ApiModelProperty(value = "The customer id that owns this shopping list.")
```

Comment:
Same comment here as for the cart ... I thought this issue was more for the `customerId` parameters in the RESP API endpoints.